k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -730,7 +696,10 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             }
 
             if (mAction != Action.EDIT_DRAFT) {
-                addAddresses(mBccView, mAccount.getAlwaysBcc());
+                String alwaysBccString = mAccount.getAlwaysBcc();
+                if (!TextUtils.isEmpty(alwaysBccString)) {
+                    recipientPresenter.addBccAddresses(new Address(alwaysBccString, ""));
```

Comment:
"alwaysBcc" could contain more than one address. We should probably call `Address.parse()` on that string.